app/components/auk/accordion/item.js

```diff
@@ -23,6 +23,12 @@ export default class AccordionItem extends Component {
 
   @action
   toggleAccordion() {
-    this.isActive = !this.isActive;
+    if (!this.args.disabled) {
+      this.isActive = !this.isActive;
+
+      if (this.args.expandTask && this.isActive) {
+        this.args.expandTask.perform();
```

Comment:
@MikiDi Zou dit eventueel een `await` (+ `async` voor action naam) moeten krijgen om correct met tasks te werken?

Reply:
Dit is besproken geweest via een call en mag zo gelaten worden.